code/DMSDocumentCartExtension.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * Class DMSDocumentCartExtension adds a control
+ *
+ * @property Boolean     AllowedInCart
+ *
+ * @property DMSDocument owner
+ */
+class DMSDocumentCartExtension extends DataExtension
+{
+    private static $db = array(
+        'AllowedInCart' => 'Boolean',
+    );
+
+    public function isAllowedInCart()
```

Comment:
Do we need this? If it's a useful API then perhaps we could add a `DMSDocumentCartInterface` which has this in it